### PR TITLE
[FIX] pylint errors

### PR DESCRIPTION
**Pull request by hbrunn**
Those are caused by stricter tests in maintainers-quality-tools

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

File: account_banking/parsers/convert.py [hunk 1]
```diff
@@ -51,10 +51,12 @@ def date2date(datestr, fromfmt='%d/%m/%y', tofmt='%Y-%m-%d'):
           "/-?:().,'+ ")
 
 
-def to_swift(astr, schemes=['utf-8', 'latin-1', 'ascii']):
+def to_swift(astr, schemes=None):
```
Comment (gurneyalex) [inline account_banking/parsers/convert.py]: side note: you could have use a tuple here instead of `None`

File: account_banking/record.py [hunk 2]
```diff
@@ -133,7 +133,9 @@ def format(self, value):
 class RecordType(object):
     fields = []
 
-    def __init__(self, fields=[]):
+    def __init__(self, fields=None):
```
Comment (gurneyalex) [inline account_banking/record.py]: same remark : a non mutable tuple would have done the trick.

File: account_banking_iban_lookup/online.py [hunk 3]
```diff
@@ -198,101 +197,3 @@ def bank_info(bic):
...
-        country_id=full_bic[4:6],
-    )
-    if '  ' in bankaddress.street:
-        bankaddress.street, bankaddress.street2 = [
-            x.strip() for x in bankaddress.street.split('  ', 1)
-        ]
-    else:
-        bankaddress.street2 = ''
-
-    return bankinfo, bankaddress
```
Comment (gurneyalex) [inline account_banking_iban_lookup/online.py:298]: @hbrunn Were you intending to commit this change? It seems unrelated to the issues flagged by Pylint.

Comment (hbrunn) [inline account_banking_iban_lookup/online.py:298]: no, it's unreachable code due to the unconditional return None, None before that block

**Comment by hbrunn** [comment]
@gurneyalex thanks for your pointer concerning tuples!

**Comment by pedrobaeza** [comment]
:+1:

**Comment by CSester** [comment]
The changes seems good. It would just be great to tell that there are not only pylint changes !
:+1:

**Comment by hbrunn** [comment]
That's what pylint says about this part of the code:

```
#run pylint command
pylint --rcfile=${PYLINT_CONFIG_DIR}/travis_run_pylint.cfg ${MODULES_TO_TEST}
[...]
************* Module banking.account_banking_iban_lookup.online
account_banking_iban_lookup/online.py:202: [W0101(unreachable), bank_info.harvest] Unreachable code
[...]
```

**Comment by CSester** [comment]
Ok, didn't see it in the Travis output.
So, :+1: for me

**Comment by StefanRijnhart** [comment]
Thanks! You beat me to it but I arrived at the same changes. :+1:
